eng/pipelines/templates/stages/archetype-js-release.yml

```diff
@@ -199,6 +199,7 @@ stages:
           artifact: ${{parameters.ArtifactName}}
           timeoutInMinutes: 5
         - ${{ each artifact in parameters.Artifacts }}:
+          - ${{if ne(artifact.options.skipPublishDevFeed, 'true')}}:
```

Comment:
Lets go ahead and set this in the tables ci.yml as well. Of course we should test this with the template project first to ensure it works.

Reply:
yes i will test this for templates